site/app/templates/submission/homework/DaylightSavingsMessage.twig

```diff
@@ -0,0 +1,11 @@
+{% if daylight %}
+    <div class="content daylight" id="daylight" style="margin-bottom: 1em; background-color: var(--standard-light-gray); color: var(--default-black)">
+        <p>
+            WARNING: Daylight savings is THIS WEEK! Please note that a late day is 24 HOURS.
+            <br>
+            This means that if an assignment is due at 10 pm a day BEFORE daylight savings in the spring, and you plan to use a late day, you must submit your assignment before 9 pm.
+            <br>
+            As for when an assignment is due at 10 pm a day AFTER daylight savings ends in the fall, you must submit your assignment by 11 pm.
```

Comment:
This is a suggestion.  Feel free to adjust (or not) as you see fit.

```suggestion
            WARNING: Daylight savings is THIS WEEK! Please note that a late day is 24 HOURS.  This means that if an assignment is due at 10 pm a day BEFORE daylight savings in the spring, and you plan to use a late day, you must submit your assignment before 9 pm.  If an assignment is due at 10 pm a day AFTER daylight savings ends in the fall, you must submit your assignment by 11 pm.
```

Reply:
This will most likely be changed soon from another issue I am working on, and I liked the old indentation better.